src/main/scala/strawman/collection/mutable/Iterator.scala

```diff
@@ -4,9 +4,10 @@ import scala.{Boolean, Int, Unit, Nothing, NoSuchElementException}
 import strawman.collection.{IndexedView, IterableOnce}
 
 /** A core Iterator class */
-trait Iterator[+A] { self =>
+trait Iterator[+A] extends IterableOnce[A] { self =>
```

Comment:
It's used implicitly in that lots of operations use `IterableOnce` parameters (analogous to `TraversableOnce in the current collections), and the idea is that both an Iterator and an Iterable are valid arguments for this.